examples/package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "sprotty-examples",
   "version": "0.13.0",
-  "private": "true",
```

Comment:
Why did you remove this? We haven't published the examples to the npm registry yet.

Reply:
It happened automatically when I ran `yarn install` oddly enough. I will add it back